apps/components-e2e/src/components/filter-field/filter-field.e2e.ts

```diff
@@ -17,20 +17,24 @@
 // tslint:disable no-lifecycle-call no-use-before-declare no-magic-numbers
 // tslint:disable no-any max-file-line-count no-unbound-method use-component-selector
 
+import { Selector } from 'testcafe';
+import { resetWindowSizeToDefault, waitForAngular } from '../../utils';
 import {
-  clickOption,
-  errorBox,
-  input,
   clearAll,
+  clickOption,
   filterTags,
   focusFilterFieldInput,
   getFilterfieldTags,
-  tagOverlay,
+  multiSelectApply,
+  multiSelectPanel,
   setupSecondTestScenario,
+  switchToSecondDatasource,
+  clickMultiSelectOption,
+  input,
+  errorBox,
+  tagOverlay,
   filterFieldRangePanel,
 } from './filter-field.po';
-import { Selector } from 'testcafe';
-import { waitForAngular, resetWindowSizeToDefault } from '../../utils';
```

Comment:
This imports have primarily resorted. It would be great to disable automatic import sorting. These changesets are really hard to grasp during a review. Maybe you can clean them up?